test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'test_helper'
+
+class RemoteCheckoutV2Test < Test::Unit::TestCase
+  def setup
+    @gateway = CheckoutV2Gateway.new(fixtures(:checkout_v2))
+
+    @amount = 200
+    @credit_card = credit_card('4242424242424242', verification_value: '100', month: '6', year: '2018')
+    @declined_card = credit_card('4000300011112220')
+
+    @options = {
+      order_id: '1',
+      billing_address: address,
+      description: 'Purchase',
+      email: "longbob.longsen@example.com"
+    }
+  end
+
+  def test_transcript_scrubbing
+    transcript = capture_transcript(@gateway) do
+      @gateway.purchase(@amount, @declined_card, @options)
+    end
+    transcript = @gateway.scrub(transcript)
+    assert_scrubbed(@declined_card.number, transcript)
+    assert_scrubbed(@declined_card.verification_value, transcript)
+    assert_scrubbed(@gateway.options[:secret_key], transcript)
+  end
+
+  def test_successful_purchase
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'Succeeded', response.message
+  end
+
```

Comment:
I'm thinking it would be useful to have a `test_successful_purchase_with_minimal_options` to make sure we don't need `order_id`, `email`, or `description` to run run a purchase.